src/flowchem/client/component_client.py

```diff
@@ -26,4 +26,11 @@ def post(self, url, data=None, json=None, **kwargs):
 
     def put(self, url, data=None, **kwargs):
         """Send a PUT request. Returns :class:`Response` object."""
+
+        # Inspect the keywargs type to avoid ploblems with not str parameters
+        if kwargs["params"]:
+            for key, arg in kwargs["params"].items():
+                if type(arg) is not str:
+                    kwargs["params"][key] = str(arg)
```

Comment:
did you make sure that int type is still handled properly?

Reply:
if lists create the problem, why not a targeted fixing of exactly this type -> would avoid potential side-effects

Reply:
targeted fixing of exactly this type!